### PR TITLE
Remove phpunit whitelist exceptions

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targeting this branch, because this is pedantic.

## Subject

I think coverage is not calculated correctly, lets see

### CODE REVIEW

File: DependencyInjection/SonataUserExtension.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ public function load(array $configs, ContainerBuilder $container)
             $authorizationCheckerReference = new Reference('security.context');
         }
 
-        // NEXT_MAJOR: Remove following lines
+        // NEXT_MAJOR: Remove following lines.
```
Comment (jordisala1991) [inline DependencyInjection/SonataUserExtension.php]: This is to trigger build

File: phpunit.xml.dist [hunk 2]
```diff
@@ -12,23 +12,26 @@
          bootstrap="Tests/bootstrap.php"
 >
     <testsuites>
-        <testsuite name="User Bundle Test Suite">
-            <directory>./Tests/</directory>
+        <testsuite name="UserBundle Test Suite">
```
Comment (greg0ire) [inline phpunit.xml.dist]: "UserBundle" isn't a thing, I think it was better before. "SonataUserBundle" is our user bundle.

Comment (jordisala1991) [inline phpunit.xml.dist]: I will change all phpunit config files. Because none of them include word Sonata. I agree with you on the namings

Comment (jordisala1991) [inline phpunit.xml.dist]: Changed to SonataUserBundle

**Comment by jordisala1991** [comment]
Can we merge this?

**Comment by greg0ire** [comment]
Sure

**Pull request merged**